akka-projection-alpakka-kafka/src/test/scala/akka/projection/scaladsl/kafka/CommittableSourceAtLeastOnceSpec.scala

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.projection.scaladsl.kafka
+
+import akka.kafka.Subscriptions
+import akka.kafka.scaladsl.Consumer
+import akka.projection.testkit.{ TestEventHandler, TestInMemoryRepository }
+import akka.stream.testkit.scaladsl.StreamTestKit.assertAllStagesStopped
+import org.scalatest.concurrent.PatienceConfiguration.Timeout
+
+import scala.concurrent.duration._
+
+class CommittableSourceAtLeastOnceSpec extends TestcontainersKafkaSpec {
+
+  implicit val patience: PatienceConfig = PatienceConfig(30.seconds, 500.millis)
+
+  "Kafka Projection using at-least-once delivery" must {
+
+    "deliver events to ProjectionHandler" in assertAllStagesStopped {
+
+      val Messages = "abc" :: "def" :: "ghi" :: Nil
+      val topic1 = createTopic(1)
+      val group1 = createGroupId(1)
+      produceString(topic1, Messages).futureValue(Timeout(remainingOrDefault))
+
+      val repository = new TestInMemoryRepository[String]
+
+      val projection =
+        KafkaProjection.atLeastOnce(
+          Consumer.committableSource(consumerDefaults.withGroupId(group1), Subscriptions.topics(topic1)),
+          committerDefaults.withMaxBatch(1),
+          TestEventHandler(repository).asAsyncHandler)
```

Comment:
Usage of at-least-once kafka projection. 

User pass a Source, the committer settings and a handler.